packages/fondue/src/components/Modal/Modal.tsx

```diff
@@ -46,13 +46,11 @@ const widthMap: Record<ModalWidth, string> = {
     [ModalWidth.Large]: 'tw-max-w-[1200px]',
 };
 
-const DEFAULT_ZINDEX = 50;
```

Comment:
Really cool! I remember few places where I had to hardcode `zIndex=0` in the web-app